packages/scene-composer/src/components/panels/scene-settings/SceneTagSettingsEditor.tsx

```diff
@@ -96,45 +96,55 @@ export const SceneTagSettingsEditor: React.FC = () => {
 
   return (
     <React.Fragment>
-      <FormField label={intl.formatMessage({ defaultMessage: 'Scale', description: 'Form Field label' })}>
-        <Grid gridDefinition={[{ colspan: 6 }, { colspan: 6 }]}>
-          <Input
-            value={String(internalScale)}
-            type='number'
-            onFocus={() => setFocusInput(true)}
-            onBlur={onInputBlur}
-            onChange={onInputChange}
-          />
-          <Checkbox
-            checked={tagSettings.autoRescale}
-            onChange={(e) => updateSettings({ autoRescale: e.detail.checked })}
-          >
-            {intl.formatMessage({ defaultMessage: 'Fixed scaling', description: 'checkbox label' })}
-          </Checkbox>
-        </Grid>
+      <SpaceBetween size='s'>
+        <FormField label={intl.formatMessage({ defaultMessage: 'Scale', description: 'Form Field label' })}>
+          <Grid gridDefinition={[{ colspan: 6 }, { colspan: 6 }]}>
+            <Input
+              value={String(internalScale)}
+              type='number'
+              onFocus={() => setFocusInput(true)}
+              onBlur={onInputBlur}
+              onChange={onInputChange}
+            />
+            <Checkbox
+              checked={tagSettings.autoRescale}
+              onChange={(e) => updateSettings({ autoRescale: e.detail.checked })}
+            >
+              {intl.formatMessage({ defaultMessage: 'Fixed scaling', description: 'checkbox label' })}
+            </Checkbox>
+          </Grid>
 
-        {(focusInput || focusSlider) && (
-          <Slider
-            value={internalScale}
-            step={0.1}
-            min='0'
-            max='10'
-            onFocus={() => {
-              setFocusSlider(true);
-            }}
-            onBlur={() => {
-              setFocusSlider(false);
-            }}
-            onMouseUp={() => {
-              setDraggingSlider(false);
-            }}
-            onMouseDown={() => {
-              setDraggingSlider(true);
-            }}
-            onChange={onSliderChange}
-          />
-        )}
-      </FormField>
+          {(focusInput || focusSlider) && (
+            <Slider
+              value={internalScale}
+              step={0.1}
+              min='0'
+              max='10'
+              onFocus={() => {
+                setFocusSlider(true);
+              }}
+              onBlur={() => {
+                setFocusSlider(false);
+              }}
+              onMouseUp={() => {
+                setDraggingSlider(false);
+              }}
+              onMouseDown={() => {
+                setDraggingSlider(true);
+              }}
+              onChange={onSliderChange}
+            />
+          )}
+        </FormField>
+        <Toggle
+          checked={!!tagSettings.disableOcclusion}
+          onChange={() => {
```

Comment:
it should return a `checked` value? using it to set the new disableOcclusion value will have less race condition in case disableOcclusion is changed somewhere else

Reply:
Done